src/utils/plot-templates/dark.js

```diff
@@ -3,6 +3,8 @@ Plotly templates are defined to override user-specified styles with Kedro-viz st
 More details can be found here - https://plotly.com/python/templates/
 */
 
+/* eslint-disable id-length,camelcase */
+
```

Comment:
I notice that there is no extra line when specifying eslint disabled rules in the other files - personally I am up for an extra line after the rule given that this will impact the entire file, and should be specified differently from how we specify block comments.

If we are adopting this practice of adding an extra line after specifying the disabled eslint rules, I suggest we should add this for the other files that has also specified disabled rulesets -  either way is fine, the key is just to be consistent. :)